base/complex.jl

```diff
@@ -148,11 +148,14 @@ Convert real numbers or arrays to complex. `i` defaults to zero.
 julia> complex(7)
 7 + 0im
 
-julia> complex([1, 2, 3])
+julia> complex([1, 2, 3], [4, 5, 6])
 3-element Array{Complex{Int64},1}:
- 1 + 0im
- 2 + 0im
- 3 + 0im
+ 1 + 4im
+ 2 + 5im
+ 3 + 6im
+
+ !!! compat "Julia 1.3"
```

Comment:
This should be unindented one space and moved outside of the code block. (I'd make this as a suggested change but those don't seem to work well for multi-line changes.) Basically, just move the closing triple backticks below the annotation to right above it, and remove the single leading space on this line.